packages/cfd/src/Containers/compare-cfds/compare-cfds.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
```

Comment:
folder name should `cfd-compare-accounts` or `compare-cfds`?
@hirad-deriv

Reply:
And we have to follow the same convention so all main files like .tsx and .scss one should have name
`cfd-compare-accounts`
Meanwhile the route will only be `compare-cfds`